packages/logger/src/Logger.ts

```diff
@@ -846,6 +846,7 @@ class Logger extends Utility implements LoggerInterface {
             'critical'
           >);
 
+    // TODO: make this.console protected or create getConsole/getConsoleMethod?
     this.console[consoleMethod](
       JSON.stringify(
         log.getAttributes(),
```

Comment:
Ideally, `this.console` should also be protected or someway accessible from subclass. Maybe we could add a `getConsole()` or `getConsoleMethod(logLevel: string)`?